examples/docs_snippets/docs_snippets/integrations/snowflake/full_example_with_advanced.py

```diff
@@ -0,0 +1,86 @@
+plot_iris_data = None
```

Comment:
Originally I had a section at the end of the reference page with a full example that incorporated all of the sections (like we did at the end of the tutorial). but then it started getting complicated with the example using the snowflake resource. Once we start including more features (partitions especially) i think it'll become unruly. Let me know if you think otherwise. But if we agree to not have a complete code snippet, i'll remove this file